protocol/writing-resource/containment.feature

```diff
@@ -52,3 +52,56 @@ Feature: Creating a resource using PUT and PATCH must create intermediate contai
     When method GET
     Then status 200
     And match testContainer.parseMembers(response) contains intermediateContainer.url
+
+  Scenario: PUT conflicts when creating resource turning resource into container
+    * def requestUri = rootTestContainer.url + 'dahut'
```

Comment:
I would advise against using `rootTestContainer` other than to get a test container for this feature file. There is one `rootTestContainer` for the whole test run so using it like this could lead to a clash if another test did the same. If you use the `testContainer` that was set up in the background then you are guaranteed to avoid a clash. (Applies below too)